microcosm_flask/paging.py

```diff
@@ -84,10 +85,15 @@ def to_tuples(self):
             ("offset", self.offset),
             ("limit", self.limit),
         ] + [
-            (key, self.rest[key])
+            (key, self._qs_value_for_tuple(self.rest[key]))
             for key in sorted(self.rest.keys())
         ]
 
+    def _qs_value_for_tuple(self, value):
```

Comment:
There are also `by_value` enums that should have `value.value` (such as https://github.com/globality-corp/florentino/blob/9c1a4b8a36800219e21e9a43855a2305e22145e9/florentino/resources/v1/qna_strategy_resources.py#L16)